systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -92,94 +89,6 @@ protected static JsonArray readUpgradeJson(String fileName) {
         }
     }
 
-    protected static Stream<Arguments> loadJsonUpgradeData() {
-        JsonArray upgradeData = readUpgradeJson(UPGRADE_JSON_FILE);
-        List<Arguments> parameters = new LinkedList<>();
-
-        List<TestKafkaVersion> testKafkaVersions = TestKafkaVersion.getSupportedKafkaVersions();
-        TestKafkaVersion testKafkaVersion = testKafkaVersions.get(testKafkaVersions.size() - 1);
-
-        upgradeData.forEach(jsonData -> {
-            JsonObject data = (JsonObject) jsonData;
-
-            data.put("urlTo", "HEAD");
-            data.put("toVersion", "HEAD");
-            data.put("toExamples", "HEAD");
-
-            // Generate procedures for upgrade
-            JsonObject procedures = new JsonObject();
-            procedures.put("kafkaVersion", testKafkaVersion.version());
-            procedures.put("logMessageVersion", testKafkaVersion.messageVersion());
-            procedures.put("interBrokerProtocolVersion", testKafkaVersion.protocolVersion());
-            data.put("proceduresAfterOperatorUpgrade", procedures);
-
-            parameters.add(Arguments.of(data.getString("fromVersion"), "HEAD", data));
-        });
-
-        return parameters.stream();
-    }
```

Comment:
removing this will completely break upgrade tests, you should check what you are deleting